### PR TITLE
[release]20210504

### PR DESCRIPTION
**Pull request by jiqiang90**

### CODE REVIEW

**Review by zhex — COMMENTED** [review]
please also pump the version of the validator package, due to the message changed.

**Review by zhex — APPROVED** [review]

File: docs/create/graphql.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ We currently supporting flowing scalars types:
 
 ## Indexing by Non-primary-key field
 
-To improve query performance, index an entity field simply by implement the `@index` annotation on a non-primary-key field.
+To improve query performance, index an entity field simply by implement the `@index` annotation on a non-primary-key field. We doesn't allow user to add `@index` annotation on any [JSON](#json-type) object. By default, we automatically add indexes on JSON fields in the database, but only to enhance query service performance.
```
Comment (ianhe8x) [inline docs/create/graphql.md:39]: `We doesn't` -> `We don't`

File: docs/create/mapping.md [hunk 2]
```diff
@@ -81,4 +81,33 @@ And these are the interface we do **NOT** support currently:
...
+
+Please note this is an **experimental features** and you may encounter bugs or issues that many negatively impact on your mapping functions. Please report any bugs you find by creating an issue in [GitHub](https://github.com/subquery/subql).
+### Built-in modules 
+
+Currently, we allow the following NodeJS modules: `assert`, `buffer`, `crypto`, `util`, and `path` . 
+
+Rather than importing the whole module, we recommend only importing the required method(s) that you need. Some methods in these modules may have dependencies that are unsupported and will fail on import. 
+
+```ts
+import {createHash} from "crypto"; // Only import required method(s)
```
Comment (ianhe8x) [inline docs/create/mapping.md:98]: this is not a good example for `Only import required method`
can use `import {hashMessage} from "ethers/lib/utils"`

File: packages/cli/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.9.3] - 2021-05-04
+### Added
+-  Indexing is automatically added into JSON field types in database to speed up query performance. Users should not add the `@index` annotation to any `@jsonField` as it will be automatically managed during the codegen process (#291)
```
Comment (ianhe8x) [inline packages/cli/CHANGELOG.md:11]: Codegen will support indexed jsonb fields. No get methods will be created for such fields in the entity class.

File: packages/node/CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.12.3] - 2021-05-04
+### Added
+- Automatically verifies that a model's indexed fields are supported by extracting indexed fields from the database (#289)
+- [Experimental Feature] We're removed the restrictions on using third party CommonJS libraries in your SubQuery project sandbox - please read more about this in our [updated documentation](https://doc.subquery.network/create/mapping.html##modules-and-libraries) (#292)
+- Support for more NodeJS modules (`assert`, `buffer`, `crypto`, `util`, `events`, and `path`) (#294)
```
Comment (ianhe8x) [inline packages/node/CHANGELOG.md:13]: `assert` was in the whitelist before. the others are newly added

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**